src/popup/pages/Networks.vue

```diff
@@ -52,9 +60,17 @@ export default defineComponent({
       deleteCustomNetwork,
     } = useNetworks();
 
+    async function switchNetworkHandler(payload: any) {
+      switchNetwork(payload);
+      if (IS_MOBILE_APP) {
+        window.location.reload();
+      }
```

Comment:
Why we need to reload the app? Maybe add some comment here?

Reply:
Turns out this is not needed anymore